content/docs/tasks/security/mtls-migration/index.md

```diff
@@ -0,0 +1,150 @@
+
+    ```command
+    $ kubectl create ns foo
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@) -n foo
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@) -n foo
+    $ kubectl create ns bar
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@) -n bar
+    $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@) -n bar
+    $ kubectl create ns legacy
+    $ kubectl apply -f @samples/sleep/sleep.yaml@ -n legacy
+    ```
+
+* Verify setup by sending an http request (using curl command) from any sleep pod (among those in namespace `foo`, `bar` or `legacy`) to `httpbin.foo`.  All requests should success with HTTP code 200.
+
+    ```command
+    $ for from in "foo" "bar" "legacy"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "sleep.${from} to httpbin.foo: %{http_code}\n"; done
+    sleep.foo to httpbin.foo: 200
+    sleep.bar to httpbin.foo: 200
+    sleep.legacy to httpbin.foo: 200
+    ```
+
+* Also verify that there are no authentication policy or destination rule in the system
+
+    ```command
+    $ kubectl get policies.authentication.istio.io --all-namespaces
+    No resources found.
+    $ kubectl get destionationrule --all-namespaces
+    No resources found.
+    ```
+
+## Configure the server to accept both mTLS and plain text traffic
+
+In Authentication Policy, we have a `PERMISSIVE` mode which makes the server accept both mutual TLS and plain text traffic.
+We need to configure the server to this mode.
+
+```bash
+cat <<EOF | istioctl create -n foo -f -
+apiVersion: "authentication.istio.io/v1alpha1"
+kind: "Policy"
+metadata:
+  name: "example-httpbin-permissive"
+  namespace: foo
+spec:
+  targets:
+  - name: httpbin
+  peers:
+  - mtls:
+      mode: PERMISSIVE
+EOF
+```
+
+Now send traffic to `httpbin.foo` again to ensure all requests can still succeed.
+
+```command
+$ for from in "foo" "bar" "legacy"; do kubectl exec $(kubectl get pod -l app=sleep -n ${from} -o jsonpath={.items..metadata.name}) -c sleep -n ${from} -- curl http://httpbin.foo:8000/ip -s -o /dev/null -w "sleep.${from} to httpbin.foo: %{http_code}\n"; done
+200
+200
+200
+```
+
+## Configure clients to send mutual TLS traffic
+
+Configure Istio services to send mutual TLS traffic by setting `DestinationRule`.
+
+```bash
+cat <<EOF | istioctl create -n foo -f -
+apiVersion: "networking.istio.io/v1alpha3"
+kind: "DestinationRule"
+metadata:
+  name: "example-httpbin-istio-client-mtls"
+spec:
+  host: httpbin.foo.svc.cluster.local
+  trafficPolicy:
+    tls:
+      mode: ISTIO_MUTUAL
+EOF
+```
+
+`sleep.foo` and `sleep.bar` should start sending mutual TLS traffic to `httpbin.foo`.
```

Comment:
"and sleep.legacy still sends http traffic to httpbin.foo since it does not have sidecar and the DestinationRule does not apply".

Reply:
Done.